src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
 // write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('table');
+  const rows = table.rows;
+
+  for (let i = 0; i < rows.length; i++) {
+    const cells = rows[i].cells;
+    const secondColumnCell = cells[1].cloneNode(true);
+
+    rows[i].insertBefore(secondColumnCell, cells[cells.length]);
```

Comment:
The `insertBefore` method is used correctly to insert the cloned cell, but the second argument should be `cells[cells.length - 1]` instead of `cells[cells.length]` to ensure the cell is inserted before the last cell in the row. Using `cells[cells.length]` will result in an error because it refers to a non-existent cell.